src/app/server/AppDelegate.h

```diff
@@ -29,7 +29,7 @@ class AppDelegate
 public:
     virtual ~AppDelegate() {}
     /**
-     * This is called on start of session establishment process
+     * This is called on start of session establishment process i.e. when the PBKDFParamRequest is received
      */
```

Comment:
in the interest of making the code base better, can't we just add docs to the other functions as well? The documentation becomes more useful in the more shared APIs like this one.

Reply:
Addressing this and the comment below in this new PR: #28138